proposals/gc/MVP.md

```diff
@@ -609,31 +609,33 @@ Casts work for both abstract and concrete types. In the latter case, they test i
 
 * `ref.test <reftype>` tests whether a reference has a given type
   - `ref.test rt : [rt'] -> [i32]`
-    - iff `rt <: trt` and `rt' <: trt` for some `trt`
+    - iff `rt <: rt'`
```

Comment:
Does this change mean that:
```
(func
  (param arrayref)
  local.get 0
  ref.test structref
  drop
)
```

is not valid because `structref </: arrayref`? Which seems to contradict that downcasts among sibling hierarchies are allowed (written later in this doc).

Reply:
Don't worry, it's just a simplification of the rule that I realised is possible and that should have no effect on what's allowed. Basically, it was redundant to distinguish between `rt'` and `trt` here before, that was only necessary for the branching rules. Your example is fine, because the rule can always pick `anyref` for `rt'`, which would be the principal type.

Reply:
Ah, yes that makes sense. Thanks.